src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/createbundle/CreateBundleAboutToSubmitHandler.java

```diff
@@ -118,6 +127,20 @@ private void setDocumentFileNameIfNotSet(SscsCaseData sscsCaseData) {
         setDocumentFileNameOnSscsDocuments(sscsCaseData);
     }
 
+    private boolean isBundleAdditionWarning(SscsCaseData sscsCaseData) {
+        List<String> bundleAdditions = emptyIfNull(sscsCaseData.getSscsDocument())
+            .stream()
+            .filter(document -> document.getValue() != null)
+            .map(document -> document.getValue().getBundleAddition())
+            .filter(bundleAddition -> bundleAddition != null)
+            .map(String::toUpperCase)
+            .collect(Collectors.toList());
+
+        return emptyIfNull(bundleAdditions).stream()
```

Comment:
I need the collection variable inside frequency function.

Reply:
Ok, can it be done with one stream? Here is a [link](https://stackoverflow.com/questions/36261036/implementation-of-having-count-with-group-by-in-java-8) I found that suggests it might be possible.

Reply:
I have used single stream now